### PR TITLE
pages/*: fix small typos

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/ssh-copy-id.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 - Copy your keys to the remote machine:
 
-`ssh-copy-id {{username@remote_host}}`
+`ssh-copy-id {{username}}@{{remote_host`
```
Comment (kbdharun) [inline pages/common/ssh-copy-id.md:8]:
```suggestion
`ssh-copy-id {{username}}@{{remote_host}}`
```

**Review by kbdharun — APPROVED** [review]

File: pages/linux/zathura.md [hunk 2]
```diff
@@ -10,15 +10,15 @@
...
+`{{H|J|K|L}} OR {{arrow keys}}`
 
 - Rotate:
 
 `r`
 
 - Invert Colors:
 
-`Ctrl + R`
+`{{Ctrl + R}}`
```
Comment (acuteenvy) [inline pages/linux/zathura.md:21]: Why?
```suggestion
`Ctrl + R`
```

**Review by gutjuri — APPROVED** [review]
Thanks

File: pages/common/pip-install.md [hunk 3]
```diff
@@ -13,7 +13,7 @@
 
 - Install packages listed in a file:
 
-`pip install --requirement {{path/to/requirements.txt}}`
+`pip install -r {{path/to/requirements.txt}}`
```
Comment (acuteenvy) [inline pages/common/pip-install.md:16]: Why change the long option for the short one?

File: pages/osx/icalbuddy.md [hunk 4]
```diff
@@ -17,7 +17,7 @@
 
 - Show tasks for a specified number of days:
 
-`icalBuddy --includeOnlyEventsFromNowOn "tasksDueBefore:today+{{8}}"`
+`icalBuddy --includeOnlyEventsFromNowOn "tasksDueBefore:today+{{days}}"`
```
Comment (acuteenvy) [inline pages/osx/icalbuddy.md:20]:
```suggestion
`icalBuddy --includeOnlyEventsFromNowOn "tasksDueBefore:today+{{number_of_days}}"`
```

**Review by acuteenvy — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**